tests/unit/testplan/testing/test_pytest.py

```diff
@@ -5,42 +5,45 @@
 import pytest
 
 from testplan.testing import py_test as pytest_runner
+from testplan.testing import ordering
+from testplan.testing import filtering
 from testplan import defaults
 from testplan import report
 
 from tests.unit.testplan.testing import pytest_expected_data
 
+PYTEST_DEFAULT_PARAMS = {
+    "test_filter": filtering.Filter(),
+    "test_sorter": ordering.NoopSorter(),
+    "stdout_style": defaults.STDOUT_STYLE,
+}
+
 
 @pytest.fixture
-def pytest_test_inst(repo_root_path, root_directory):
```

Comment:
I remember root_directory is needed for internal test.

Reply:
yes. this is restored.